packages/fuels-core/src/constants.rs

```diff
@@ -15,3 +15,5 @@ pub const BASE_ASSET_ID: AssetId = AssetId::new([0u8; 32]);
 pub const DEFAULT_GAS_ESTIMATION_TOLERANCE: f64 = 0.2;
 pub const GAS_PRICE_FACTOR: u64 = 1_000_000_000;
 pub const MAX_GAS_PER_TX: u64 = 100_000_000;
+
+pub const FAILED_TRANSFER_TO_OUTPUT_SIGNAL: u64 = 0xffff_ffff_ffff_0001;
```

Comment:
Can you add a comment explaining what this means? Is this some kind of error code from `fuel-core`?